drivers/storage/portworx/deployment.go

```diff
@@ -206,6 +223,12 @@ func (p *portworx) GetKVDBPodSpec(
 		return v1.PodSpec{}, err
 	}
 
+	if node, err := p.getNodeByName(nodeName); err != nil {
+		logrus.WithError(err).Warnf("Could not get OSImage for node %s", nodeName)
+	} else {
+		t.setOSImage(node.Status.NodeInfo.OSImage)
+	}
+
```

Comment:
Do we need this here? We don't do anything for the kvdb pod wrt bottle rocket.

Reply:
@piyush-nimbalkar -- I thought this was applied when the POD actually _includes_ the KVDB deployment (so, PX/OCI-Monitor + KVDB), in which case it would be needed.
However, if this deploys only the KVDB (without PX), then this might introduce a bug  (i.e. etcd getting extra `-disable-log-proxy --install-uncompress` args would fail to start).

Could you advise how to test this code-path?

Reply:
The `GetKVDBPodSpec` is called by [StorageNode controller](https://github.com/libopenstorage/operator/blob/master/pkg/controller/storagenode/storagenode.go#L322), where it just creates place holder pods where internal kvdb runs. It is useful for pod disruption budgets.

The above code you added is kind of a dead code as the `osImage` field won't be used to construct the kvdb container at all. It is only used to construct portworx pod which is created only in `GetPortworxPodSpec`.